e2e-test/workspace_helper.py

```diff
@@ -126,4 +183,36 @@ def share_workspace_grant_owner(orch_url, billing_project_name, workspace_name,
     if status_code != 200:
         raise Exception(response.text)
 
-    logging.info(f"Successfully shared workspace {workspace_name} with {email_to_share}")
+    logging.info(f"Successfully shared workspace {workspace_name} with {email_to_share} as OWNER")
+
+
+# ADD WDL TO GCP WORKSPACE ACTION
+def add_wdl_to_gcp_workspace(billing_project_name, workspace_name, wdl_namespace, wdl_name, method_definition_dict, root_entity_type, inputs_dict, outputs_dict, orch_url, token):
+    uri = f"{orch_url}//api/workspaces/{billing_project_name}/{workspace_name}/method_configs/{wdl_namespace}/{wdl_name}"
+    request_body = {
+        "namespace": wdl_namespace,
+        "name": wdl_name,
+        "rootEntityType": root_entity_type,
+        "workspaceName": {
+            "name": workspace_name,
+            "namespace": billing_project_name
+        },
+        "methodRepoMethod": method_definition_dict,
+        "outputs": outputs_dict,
```

Comment:
im curious if you actually need these inputs/outputs.  like if this was empty and then the service "fixed" it when trying to submit, would we have a functioning method config then?

I only ask because then we wouldnt have to worry about updating this if the wdl inputs/outputs change, the service would do that for us

Reply:
good thinking, it does still work without specifying them. that's good from the point of view of using with our service, but less good from the point of view of using this workspace for other purposes. i wonder if we should keep these here, but when we set up the e2e script we should not specify them so we don't have to update the e2e script if the inputs/outputs change?

Reply:
gotcha that makes sense